src/React/doc/index.rst

```diff
@@ -89,10 +89,25 @@ For example:
     }
 
 
+As ``react_component`` uses `Encore`_, it will need the two following twig lines to work
+
+.. code-block:: twig
+
+    {% block stylesheets %}{{ encore_entry_link_tags('app') }}{% endblock %}
+    {% block javascripts %}{{ encore_entry_script_tags('app') }}{% endblock %}
+
+They should already be in the base.html.twig template, so we can just extend it
+
 .. code-block:: html+twig
 
     {# templates/home.html.twig #}
-    <div {{ react_component('MyComponent', { 'fullName': app.user.fullName }) }}></div>
+    {% extends 'base.html.twig' %}
+
+    {% block body %}
+        <div {{ react_component('MyComponent', { 'fullName': number }) }}>
+            Loading... <i class="fas fa-cog fa-spin fa-3x"></i>
+        </div>
+    {% endblock %}
```

Comment:
I'm totally fine with this change - it makes it more complete, without distracting too much